public/testem/qunit_adapter.js

```diff
@@ -5,8 +5,8 @@ qunit_adapter.js
 
 Testem's QUnit adapter. Works by using QUnit's hooks:
 
-* `testStart`
-* `testDone`
+* `begin`
+* `done`
```

Comment:
these changes might break.

`testStart` and `testDone` are hooks for before and after each test block, while `begin` and `done` stands for the entire suite.